custom/icds_reports/utils/aggregation_helpers/distributed/agg_awc.py

```diff
@@ -35,6 +36,11 @@ def child_temp_tablename(self):
     def agg_child_temp_tablename(self):
         return get_agg_child_temp_tablename()
 
+    def get_table(self, table_id):
+        if self.month_start != transform_day_to_month(date.today()):
+            return get_prev_table(table_id)
```

Comment:
what's the human-readable explanation of this check?

Reply:
"Is this the current month agg?" is the human readable version but will pull it into a better named function